### DISCUSSION TITLE
Ubuntu 24.04 LTS and Thunderbird provided as Snap - Plan for Linux Mint 22 and next versions

### DISCUSSION THREAD

**Discussion by Angedestenebres** [linuxmint/discussions#320] [Ideas]
Hello,

I have a bad news for Linux Mint 22, Ubuntu continues with their snapization and the next candidate is Thunderbird, see: https://news.itsfoss.com/thunderbird-snap/
It's planned for the next stable release, that means for the LTS 24.04 and we will have an issue on Linux Mint due to that.

Do you have a plan to include it in Linux Mint repo like Firefox? Or do you use the [flatpak](https://flathub.org/fr/apps/org.mozilla.Thunderbird) instead?

Thanks in advance for your answer.

**Comment by mtwebster** [comment]
We'll package it ourselves like Firefox. Someone pointed this out yesterday, I've already started working on the build.

**Reply by Angedestenebres** [reply]
Top :+1: 
Thanks for the quick answer and your work.
